Jenkinsfile

```diff
@@ -101,6 +101,7 @@ pipeline {
                         dir('src/github.com/docker/app') {
                             unstash 'binaries'
                             unstash 'e2e'
+			    sh 'ls -la .'
```

Comment:
tabs -> spaces, but do we need this? It looks like a debug command left over?

Reply:
yes just a debug to get why the CI failed sometimes :wink: 
I'll remove :stuck_out_tongue_closed_eyes:

Reply:
Updated :wink: